### PR TITLE
rename visible => open everywhere

### PR DESCRIPTION
**Pull request by tomquirk**
KILL THE VISIBLE PROP.

Got sick of the console errors. one fell swoop cleanup

### CODE REVIEW

File: src/components/v2v3/V2V3Project/V2V3ManageTokensSection/index.tsx [hunk 1]
```diff
@@ -202,7 +202,8 @@ export default function V2ManageTokensSection() {
                 >
                   <div>
                     <div>
-                      {totalTokenSupplyDiscrete} {tokenText}
+                      {formatWad(totalTokenSupply, { precision: 0 })}{' '}
```
Comment (tomquirk) [inline src/components/v2v3/V2V3Project/V2V3ManageTokensSection/index.tsx:205]: unrelated fix

File: src/hooks/GnosisSafe.ts [hunk 2]
```diff
@@ -24,6 +24,7 @@ export const useGnosisSafe = (address?: string) => {
     },
     {
       retry: false,
+      refetchOnWindowFocus: false,
```
Comment (tomquirk) [inline src/hooks/GnosisSafe.ts:27]: unrelated improvement.

**Comment by johnnyd-eth** [comment]
Yep scrolled through the code and gave a decent smoke test - LGTM. Great cleanup 🙌

**Review by johnnyd-eth — APPROVED** [review]
